spec/controllers/portal/client_logins_controller_spec.rb

```diff
@@ -1,9 +1,11 @@
 require "rails_helper"
+include ActiveSupport::Testing::TimeHelpers
 
 RSpec.describe Portal::ClientLoginsController, type: :controller do
   let(:client) do
     create(
       :client,
+      last_seen_at: DateTime.new(2021, 4, 20, 16, 21),
```

Comment:
If this is intended to be an arbitrary time in the past, consider setting it to `nil` instead.